frontend/src/metabase/parameters/components/ParameterLinkedFilters/ParameterLinkedFilters.tsx

```diff
@@ -111,6 +101,42 @@ const ParameterLinkedFilters = ({
   );
 };
 
+function getDisableReason({
+  usableParameters,
+  parameter,
+  onShowAddParameterPopover,
+}: {
+  usableParameters: Parameter[];
+  parameter: Parameter;
+  onShowAddParameterPopover: () => void;
+}): JSX.Element | null {
+  if (usableParameters.length === 0) {
+    return (
+      <div>
+        <SectionMessage>
+          {t`If you have another dashboard filter, you can limit the choices that are listed for this filter based on the selection of the other one.`}
+        </SectionMessage>
+        <SectionMessage>
+          {jt`So first, ${(
+            <SectionMessageLink key="link" onClick={onShowAddParameterPopover}>
+              {t`add another dashboard filter`}
+            </SectionMessageLink>
+          )}.`}
+        </SectionMessage>
+      </div>
+    );
+  } else if (parameter.values_source_type != null) {
```

Comment:
No need to put `else` after `return`.

```suggestion
  }

  if (parameter.values_source_type != null) {
```